ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionengine/ExecutionEngineChannel.java

```diff
@@ -52,6 +52,12 @@
         public SafeFuture<PayloadStatus> newPayload(final ExecutionPayload executionPayload) {
           return SafeFuture.completedFuture(PayloadStatus.SYNCING);
         }
+
+        @Override
+        public SafeFuture<TransitionConfiguration> exchangeTransitionConfiguration(
+            TransitionConfiguration transitionConfiguration) {
+          return SafeFuture.completedFuture(null);
```

Comment:
`null` is an awkward response type here.  Not sure but might be better to just return the supplied config here too.